Source/Classes/PINRemoteImageManager.m

```diff
@@ -975,10 +980,15 @@ - (NSUUID *)prefetchImageWithURL:(NSURL *)url
 }
 
 - (NSUUID *)prefetchImageWithURL:(NSURL *)url options:(PINRemoteImageManagerDownloadOptions)options
+{
+    return [self prefetchImageWithURL:url options:options priority:PINRemoteImageManagerPriorityLow];
+}
+
+- (NSUUID *)prefetchImageWithURL:(NSURL *)url options:(PINRemoteImageManagerDownloadOptions)options priority:(PINRemoteImageManagerPriority)priority
 {
     return [self downloadImageWithURL:url
```

Comment:
Would be great to refactor this method also to maybe: `downloadImageWithURL:info:` or something API. cc @garrettmoon @nguyenhuy